201-logic-app-transform-function/metadata.json

```diff
@@ -2,6 +2,6 @@
     "itemDisplayName": "Azure function with transform capabilites",
     "description": "Creates a webhook based C# azure function with transform capabilites to use in logic apps integration scenarios",
     "summary": "Creates a webhook based C# azure function with transform capabilites",
-    "githubUsername": "rarayudu",
-    "dateUpdated": "2016-06-02"
+    "githubUsername": "jaakhter",
```

Comment:
Yes, I'll be owning this template for now. There are more work to be done by me on this. Hence, changed it,